e2e-tests/run.sh

```diff
@@ -1,3 +1,2 @@
 #!/bin/bash
-
 sh e2e-tests/batch-mode-tests.sh
```

Comment:
nit: Since you have added the `#!/bin/bash` header, I think you can simply run this instead of passing through `sh`, e.g.,
```
./e2e-tests/batch-mode-tests.sh
```

Reply:
Am passing through run.sh because I intend to add other script like streaming-mode-tests.sh and I can call them once by calling run.sh

Reply:
Right, but by using `./e2e-tests/batch-mode-tests.sh`, the `batch-mode-tests.sh` script will run the script in the same shell, which is likely what we want.